app/views/layouts/application.html.haml

```diff
@@ -14,7 +14,7 @@
     %meta{:name => "author", :content => "Diaspora, Inc."}/
 
     %link{:rel => 'shortcut icon', :href => "#{image_path('favicon.png')}" }
-    %link{:rel => 'apple-touch-icon', :href => "#{image_path('apple-touch-icon.png')}"}
+    %link{:rel => 'apple-touch-icon', :href => "#{image_path('favicon.png')}"}
```

Comment:
The problem is the following: it is also used on browser for tabs favicons, but the image is way too big. The rendering looks messy. If someone has a better solution, I'm open to that. In any case, I suspect there are lots of images that aren't used anymore in the interface and that could be suppressed.